core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowResource.scala

```diff
@@ -558,15 +581,133 @@ class WorkflowResource {
     */
   def getOwnerFilter(owners: java.util.List[String]): Condition = {
```

Comment:
Is this filter using email instead of names? If so, you need to explain why in both the PR description and the above comment for this function.

Reply:
Because of #1839, this change is necessary